proposals/0332-swiftpm-command-plugins.md

```diff
@@ -414,19 +411,13 @@ In the SwiftPM CLI, command plugins provided by a package or its direct dependen
 
 This will invoke the plugin and only return when it completes. Since no other options were provided, this will pass all regular targets in the package to the plugin ("special" targets such as those that define plugins will be excluded).
 
-To pass a subset of the targets to the plugin, one or more `--target` options can be used in the invocation:
+Any parameters passed after the name of the plugin command are passed verbatim to the entry point of the plugin.  For example, if a plugin accept a `--target` option, a subset of the targets to operate on can be passed on the command line that invokes the plugin:
 
 ```shell
-❯ swift package --target Foo --target Bar do-something
+❯ swift package Bar do-something --target Foo --target Bar --someOtherFlag
```

Comment:
first "Bar" typo?

Reply:
Yes indeed.  Thanks for pointing that out.